GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml

```diff
@@ -0,0 +1,266 @@
+﻿@using GenderPayGap.Core
+@using GenderPayGap.Database.Models
+@using GenderPayGap.WebUI.Models.Admin
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model List<GenderPayGap.Database.Models.Feedback>
+
+@{
+    ViewBag.Title = $"View feedback - Administration - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+{
+            new CrumbViewModel
+            {
+                Text = "Admin",
+                Href = Url.Action("Home", "Admin")
+            },
+            new CrumbViewModel
+            {
+                Text = "View feedback"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+@{

```

Comment:
Thanks James, It looks great. Can we probably break this file to two files, one partial view FeedbackRows?

Reply:
Good idea - it looks much nicer now as two files